boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -167,6 +145,38 @@ config GPIO_DW_0
 
 endif
 
+if FLASH
+
+config SPI
+	def_bool y
+
+config SPI_NOR
+	def_bool y
+
+config SPI_NOR_JEDEC_ID
+	default 0xC2803A
+
+config SPI_NOR_PAGE_SIZE
+	default 256
+
+config SPI_NOR_SECTOR_SIZE
+	default 4096
+
+config SPI_NOR_SECTORS
```

Comment:
Number of sectors should be calculated from the flash density.

Reply:
You mean (Flash size)/(sector size)? Yes, that can be done.

Reply:
Done